### PR TITLE
Build WieneckeSinske on Linux

### PR DESCRIPTION
**Pull request by hadim**

### CODE REVIEW

**Comment by hadim** [comment]
@marktsuchida I just noticed you and @nicost are actively working on MM2 and also switching te build system to Gradle.

This is a very good news!

Please let me know when the build is ok to be tested under Linux so I can adapt this PR to it.

File: DeviceAdapters/WieneckeSinske/CAN29.cpp [hunk 1]
```diff
@@ -23,12 +23,14 @@
...
-//                INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES.  
+//                INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES.
 //
 
 #ifdef WIN32
 #include <windows.h>
-#define snprintf _snprintf 
+#define snprintf _snprintf
+#elif __linux__
+#include <arpa/inet.h>
```
Comment (kmdouglass) [inline DeviceAdapters/WieneckeSinske/CAN29.cpp:33]: Are these the only lines that are necessary to get the Linux build working? There are a lot of diffs generated by (I assume) the addition/removal of spaces.

Comment (hadim) [reply DeviceAdapters/WieneckeSinske/CAN29.cpp:33]: Yes they are. Sorry for the formatting issue!

Comment (kmdouglass) [reply DeviceAdapters/WieneckeSinske/CAN29.cpp:33]: Is there anything that I need to do beyond adding the #include statement to get the Linux build to work? I added the lines you suggest but I still run into the =error: ‘htonl’ was not declared in this scope= error. I checked and I have the inet header on my system, but maybe I need to set an environment variable before building?

EDIT

Ah sorry, I only added the lines to CAN29.cpp. I'll try adding them to WieneckeSinske.cpp as well and try again.

Comment (hadim) [reply DeviceAdapters/WieneckeSinske/CAN29.cpp:33]: With that script https://github.com/hadim/mm_scripts/blob/master/build-mm.sh you should be able to install MM from a freshly installed Ubuntu.

Comment (kmdouglass) [reply DeviceAdapters/WieneckeSinske/CAN29.cpp:33]: Thanks! The mistake was mine: I only added the include statement to one of two necessary .cpp files ;) Everything has compiled successfully now.

**Comment by nicost** [comment]
These fixes were pushed to the subversion repository in r16768 and r16767 (sorry for the long delay!).  I no longer have the ability to close PRs (or do anything else in the MM repository I guess).  Can someone close this PR?

**Pull request closed without merging**